pick.lic

```diff
@@ -203,10 +203,13 @@ class LockPicker
     end
 
     if @make_pets
-      case bput("put my #{box} in my #{@settings.picking_pet_box_source}", 'You put your .* in your ', 'You just can\'t get')
+      case bput("put my #{box} in my #{@settings.picking_pet_box_source}", 'You put your .* in your ', 'You just can\'t get', 'There isn\'t any more')
       when 'You just can\'t get'
```

Comment:
You can combine these two cases with `when 'You just can\'t get', 'There isn\'t any more'`